pom.xml

```
@@ -21,7 +21,15 @@
 		<plugin.prettier.goal>write</plugin.prettier.goal>
 		<spring-cloud.version>2021.0.3</spring-cloud.version>
 		<testcontainer.version>1.17.3</testcontainer.version>
+		<de.uni-stuttgart.gamify-it.authentification-validator>v1.0.0</de.uni-stuttgart.gamify-it.authentification-validator>
```

I think it would be nice to add `.version` here
```suggestion
		<de.uni-stuttgart.gamify-it.authentification-validator.version>v1.0.0</de.uni-stuttgart.gamify-it.authentification-validator.version>
```